pkg/interface/chat/src/js/components/lib/invite-search.js

```diff
@@ -33,18 +34,32 @@ export class InviteSearch extends Component {
     groups = groups.filter(e => !e.startsWith("/~/"));
 
     let peers = [],
-      peerSet = new Set();
+      peerSet = new Set(),
+      contacts = new Map;
     Object.keys(this.props.groups).map(group => {
       if (this.props.groups[group].size > 0) {
         let groupEntries = this.props.groups[group].values();
         for (let member of groupEntries) {
           peerSet.add(member);
         }
       }
+      if (this.props.contacts[group]) {
+        let groupEntries = this.props.groups[group].values();
+        for (let member of groupEntries) {
+          if (this.props.contacts[group][member]) {
+            if (contacts.has(member)) {
+              contacts.get(member).push(this.props.contacts[group][member].nickname);
+            }
+            else {
+              contacts.set(member, [this.props.contacts[group][member].nickname]);
```

Comment:
Looks great, the performance characteristics of this are fine. People aren't going to have 100s or 1000s of contacts for a while.